### PR TITLE
Revamping README.md

### PR DESCRIPTION
**Pull request by apowers313**
Clarifying the README.md, per the thread in #25.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,46 +16,57 @@ semantic-release-cli setup
...
-
-The CI environment has to export `CI=true` in order for `semantic-release` to not automatically perform a dry run. Most CI services do this by default.
-
-You need to export access tokens to the environment, so `semantic-release` can authenticate itself with GitHub and npm.  [Get a token for GitHub on their website](https://github.com/settings/tokens/new), grant it the repo/public_repo scope, and export it as `GH_TOKEN`. The smoothest and securest way to do this on Travis CI is to use [their web interface](http://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings).
-
-Unfortunately there is no web interface for obtaining npm tokens yet, so you have to run `npm adduser` locally. Copy the token from your `~/.npmrc` file afterwards. Export it as `NPM_TOKEN`.
+## Options
+
+	Usage:
+	  semantic-release-cli setup [--tag=<String>]
```
Comment (boennemann) [inline README.md]: `--tag=<String>` should be `options...` here :)

**Comment by gr2m** [comment]
@christophwitzko @boennemann could we get this merged? I'd then also really like to add documentation on the `--gh-token` and `--npm-token` arguments you can pass to CLI now, via #31

**Comment by christophwitzko** [comment]
LGTM :+1: 
Except for the "commitizen friendly" thing. @boennemann: What is your thought on this?

**Comment by boennemann** [comment]
Sorry for the absurd delay. Thanks a lot @apowers313 . I'll merge this now, and move the commitizen config into the package.json at a later point.

**Pull request merged**